### DISCUSSION TITLE
[Mini.extra] LSP Workspace Symbols being limited

### DISCUSSION THREAD

**Discussion by ldelossa** [nvim-mini/mini.nvim#749] [Q&A] [answered]
Hello there.

I'm working with the Linux kernel. I went to do a workspace symbols search today and I think 'Pick lsp' is limited the results from the LSP. 

Per the docs 

```
{local_opts} `(table)` Options defining behavior of this particular picker.
  Possible fields:
  - <scope> `(string)` - LSP method to use. One of the supported ones (see
    list above). Default: `nil` which means explicit scope is needed.
  - <symbol_query> `(string)` - query for |vim.lsp.buf.workspace_symbol()|.
    Default: empty string for all symbols (according to LSP specification).
{opts} `(table|nil)` Options forwarded to |MiniPick.start()|.
```

When I open with a empty string tho, it seems to be limited to 100. I'm not sure if this is a mini issue or is this an LSP limitation?

**Comment by echasnovski** [comment, marked as answer]
Hi!

I'd assume you are using `clangd`. It has `--limit-results` option for "Limit the number of results returned by clangd. 0 means no limit (default=100)." So if you want a bigger number, supply a different value during LSP server initialization. For example, if done directly (not with 'nvim-lspconfig'): `vim.lsp.start({ name = 'clangd', cmd = { 'clangd', '--limit-results=1000' }, root_dir = vim.fn.getcwd()})`

**Reply by ldelossa** [reply]
thank you so much!
